doc/configuration.md

```diff
@@ -253,6 +253,30 @@ made, is propagated between services along the trace in the form of the
 order to prevent rejection by peers or other HTTP header policies.  This
 configuration option is that limit, in bytes.
 
+### Span Sampling Rules
+When a trace is dropped, it may still be advantageous to send some of its spans
```

Comment:
Consider restructuring this without the term "advantageous".
A simpler explanation of what it does, examples of cases where it's the right thing to do and link to how to do so should be enough.